Test failure: `TestCreateReadUpdateDeleteIbaDashboards`

```
GOROOT=/usr/local/go #gosetup
GOPATH=/Users/cmarget/go #gosetup
/usr/local/go/bin/go test -c --tags=integration -tags integration -o /Users/cmarget/Library/Caches/JetBrains/GoLand2024.3/tmp/GoLand/___TestCreateReadUpdateDeleteIbaDashboards_in_github_com_Juniper_apstra_go_sdk_apstra.test github.com/Juniper/apstra-go-sdk/apstra #gosetup
/usr/local/go/bin/go tool test2json -t /Users/cmarget/Library/Caches/JetBrains/GoLand2024.3/tmp/GoLand/___TestCreateReadUpdateDeleteIbaDashboards_in_github_com_Juniper_apstra_go_sdk_apstra.test -test.v=test2json -test.paniconexit0 -test.run ^\QTestCreateReadUpdateDeleteIbaDashboards\E$ #gosetup
=== RUN   TestCreateReadUpdateDeleteIbaDashboards
2024/12/26 15:18:08 unsupported API version: '5.1.0'
=== RUN   TestCreateReadUpdateDeleteIbaDashboards/5.1.0_i-0a36daf0e2501f5e7
=== PAUSE TestCreateReadUpdateDeleteIbaDashboards/5.1.0_i-0a36daf0e2501f5e7
=== CONT  TestCreateReadUpdateDeleteIbaDashboards/5.1.0_i-0a36daf0e2501f5e7
    api_iba_dashboards_test.go:39: 18
    api_iba_dashboards_test.go:52: Name :device_prerequisites Created Id :174082be-ebaf-43c5-975c-bad9cbe6a8bf
    api_iba_dashboards_test.go:53: Getting Dashboard
    api_iba_dashboards_test.go:58: Updating Dashboard
    api_iba_dashboards_test.go:61: 
        	Error Trace:	/Users/cmarget/projects/work/apstra-go-sdk/apstra/api_iba_dashboards_test.go:61
        	Error:      	Received unexpected error:
        	            	attempt to update dashboard with non-empty updated_by value - this value can be set only by the server
        	Test:       	TestCreateReadUpdateDeleteIbaDashboards/5.1.0_i-0a36daf0e2501f5e7
--- FAIL: TestCreateReadUpdateDeleteIbaDashboards/5.1.0_i-0a36daf0e2501f5e7 (8.93s)

--- FAIL: TestCreateReadUpdateDeleteIbaDashboards (0.52s)

FAIL

Process finished with the exit code 1
```

--- a/apstra/api_iba_dashboards.go
+++ b/apstra/api_iba_dashboards.go
@@ -1,4 +1,4 @@
-// Copyright (c) Juniper Networks, Inc., 2022-2024.
+// Copyright (c) Juniper Networks, Inc., 2022-2025.
 // All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
@@ -253,11 +253,12 @@ func (o *Client) createIbaDashboard(ctx context.Context, blueprintId ObjectId, i
 
 func (o *Client) updateIbaDashboard(ctx context.Context, blueprintId ObjectId, id ObjectId, in *IbaDashboardData) error {
 	if in.UpdatedBy != "" {
-		return errors.New("attempt to update dashboard with non-empty updated_by value - this value can be set only by the server")
+		return errors.New(fmt.Sprintf("attempt to update dashboard with non-empty updated_by value - this value can be set only by the server Got '%s', Expected blank", in.UpdatedBy))
 	}
 	if in.PredefinedDashboard != "" {
-		return errors.New("attempt to update dashboard with non-empty predefined_dashboard value - this value can " +
-			"be set only by the server, and only when a dashboard is instantiated from a predefined template")
+		return errors.New(fmt.Sprintf("attempt to update dashboard with non-empty predefined_dashboard value - this value can "+
+			"be set only by the server, and only when a dashboard is instantiated from a predefined template "+
+			"Got '%s' Expected blank", in.PredefinedDashboard))
 	}
 
 	err := o.talkToApstra(ctx, &talkToApstraIn{

--- a/apstra/api_iba_dashboards_test.go
+++ b/apstra/api_iba_dashboards_test.go
@@ -1,4 +1,4 @@
-// Copyright (c) Juniper Networks, Inc., 2022-2024.
+// Copyright (c) Juniper Networks, Inc., 2022-2025.
 // All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
@@ -56,6 +56,8 @@ func TestCreateReadUpdateDeleteIbaDashboards(t *testing.T) {
 
 				d1.Data.Label = randString(5, "hex")
 				t.Log("Updating Dashboard")
+				d1.Data.UpdatedBy = ""
+				d1.Data.PredefinedDashboard = ""
 
 				err = bpClient.UpdateIbaDashboard(ctx, id, d1.Data)
 				require.NoError(t, err)

--- a/apstra/api_iba_widgets.go
+++ b/apstra/api_iba_widgets.go
@@ -1,4 +1,4 @@
-// Copyright (c) Juniper Networks, Inc., 2023-2024.
+// Copyright (c) Juniper Networks, Inc., 2023-2025.
 // All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
@@ -6,17 +6,9 @@ package apstra
 
 import (
 	"encoding/json"
-	"time"
-
 	"github.com/Juniper/apstra-go-sdk/apstra/enum"
+	"time"
 )
-
-type IbaWidget struct {
-	Id        ObjectId
-	CreatedAt time.Time
-	UpdatedAt time.Time
-	Data      *IbaWidgetData
-}
 
 type IbaWidgetData struct {
 	AggregationPeriod  *time.Duration

--- a/apstra/api_iba_widgets.go
+++ b/apstra/api_iba_widgets.go
@@ -6,8 +6,9 @@ package apstra
 
 import (
 	"encoding/json"
-	"github.com/Juniper/apstra-go-sdk/apstra/enum"
 	"time"
+
+	"github.com/Juniper/apstra-go-sdk/apstra/enum"
 )
 
 type IbaWidgetData struct {

